gitlab/const.py

```diff
@@ -27,17 +30,48 @@
 MAINTAINER_ACCESS: int = 40
 OWNER_ACCESS: int = 50
 
+
+class AccessLevel(Enum):
```

Comment:
I think using `IntEnum` might make sense here.

Reply:
Agreed that all of them should use `IntEnum` as otherwise comparisons with integer values will fail. From my reading of the documentation.